app/controllers/items_controller.rb

```diff
@@ -231,7 +231,7 @@ def edit_copyright
 
   # save the copyright form
   def update
-    change_set = ItemChangeSet.new(@cocina)
+    change_set = @cocina.is_a?(Cocina::Models::Collection) ? CollectionChangeSet.new(@cocina) : ItemChangeSet.new(@cocina)
```

Comment:
random thought, not for this PR:  should we have `cocina-obj.collection?` `.dro?` and `.apo?` convenience methods provided in cocina-models?

Reply:
I believe they already exist.

Reply:
https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/checkable.rb